src/Adapter/AbstractObjectModelFiller.php

```diff
@@ -0,0 +1,108 @@
+<?php
+/**
+ * Copyright since 2007 PrestaShop SA and Contributors
+ * PrestaShop is an International Registered Trademark & Property of PrestaShop SA
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.md.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://devdocs.prestashop.com/ for more information.
+ *
+ * @author    PrestaShop SA and Contributors <contact@prestashop.com>
+ * @copyright Since 2007 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ */
+
+declare(strict_types=1);
+
+namespace PrestaShop\PrestaShop\Adapter;
+
+use ObjectModel;
+use PrestaShop\PrestaShop\Core\Exception\CoreException;
+
+class AbstractObjectModelFiller
+{
+    /**
+     * @param ObjectModel $objectModel
+     * @param string $propertyName
+     * @param array $properties
+     */
```

Comment:
```suggestion
     * @param array $properties
     *
     * @return void
     */
```

Reply:
I don't see the interest of PHPDoc `@return void` Because to me `void` is not even a type, it returns nothing So strictly typing a method as void makes sense But the `@return` in this case has no interest and is even more confusing
Of course it's just my point of view ^^

Reply:
>  strictly typing a method as void makes sense But the @return in this case has no interest

I agree